specification/alertsmanagement/resource-manager/readme.md

```diff
@@ -44,7 +44,20 @@ directive:
 title: AlertsManagementClient
 description: AlertsManagement Client
 openapi-type: arm
-tag: package-2023-03
+tag: package-2024-05
+```
+
+### Tag: package-2024-05
+
+These settings apply only when `--tag=package-2024-05` is specified on the command line.
+
+```yaml $(tag) == 'package-2023-03'
```

Comment:
This `package-2023-03` instead of `pacakge-2024-05` triggers a LintDiff bug:

- https://github.com/Azure/azure-sdk-tools/issues/8568